### ISSUE TITLE
overflow:scroll on the container is bad

### ISSUE THREAD

**Issue by ThiefMaster** [shotaK/redux-lazy-scroll#1]
It results in scroll bars on the container even though there is nothing to scroll in it. This is already the case in the demo here: https://ancient-sands-71156.herokuapp.com/

**Comment by shotaK** [comment]
What you can do is that:

1) Get the data from the server for checking whether there are items left to fetch or the total amount of items. E.g `{total: 100}`
2) There is a `hasMore` prop. Pass there `false` if there are no items left. E.g 
```
<ReduxLazyScroll
    hasMore={store.fetchedItems.length === store.total}
>
```
This way it won't make an additional request.

**Comment by ThiefMaster** [comment]
I'm not talking about additional requests (I'm already using `hasMore`). I'm talking about these scroll bars:

![image](https://user-images.githubusercontent.com/179599/39135331-f8db4600-4718-11e8-9c90-4517161e48f3.png)

There are horizontal ones as well.

Since the scrollable element is the body/document itself, having scroll bars on the `.redux-lazy-scroll` container looks a lot like a bug.

**Comment by shotaK** [comment]
Yes, you are right, it should only have `scroll` if the prop `isParentScrollable` is set true.

**Comment by SidneyNiccolson** [comment]
Hi @shotaK,

Any fix for this? I'm currently using custom CSS to set overflow to hidden. Which is not ideal.

**Comment by himanshu64** [comment]
Hi @shotaK 
any update on this bug?
